doc/drivers/dtr_uart.rst

```diff
@@ -0,0 +1,76 @@
+.. _drivers_dtr_uart:
+
```

Comment:
The path for this driver file (drivers/dtr_uart) could be added in the index file (doc/index).